src/xrCDB/ISpatial.cpp

```diff
@@ -272,7 +272,10 @@ void ISpatial_DB::_insert(ISpatial_NODE* N, Fvector& n_C, float n_R)
 void ISpatial_DB::insert(ISpatial* S)
 {
     pcs->Enter();
+
 #ifdef DEBUG
+#ifndef LINUX // disabled for Linux because linking issue due to IGameObject belongs to xrEngine 
```

Comment:
Better hide under define "else" part an in LINUX macro branch do some this: xrDebug::Fatal(DEBUG_INFO, "Invalid PS or other spatial position (%s)", O->cName().c_str());